src/shared/telemetry/vscodeTelemetry.json

```diff
@@ -25,6 +25,23 @@
         }
     ],
     "metrics": [
+        {
+            "name": "aws_refreshCredentials",
```

Comment:
Should move this to the common definitions, although we already have like 6 definitions for telemetry. Could use `aws_loadCredentials` I suppose.

Reply:
I'm using the metric for observability purposes so there's a lot less value in sharing.

Reply:
+1 for upstreaming it. Should be passive though.